src/app/util/attribute-storage.cpp

```diff
@@ -878,6 +878,51 @@ static uint16_t findIndexFromEndpoint(EndpointId endpoint, bool ignoreDisabledEn
     return kEmberInvalidEndpointIndex;
 }
 
+uint16_t emberAfGetClusterServerEndpointIndex(EndpointId endpoint, ClusterId cluster, uint16_t fixedClusterServerEndpointCount)
+{
+    uint16_t epIndex = findIndexFromEndpoint(endpoint, true /*ignoreDisabledEndpoints*/);
+
+    // Endpoint must be configured and enabled
+    if (epIndex != kEmberInvalidEndpointIndex)
```

Comment:
Why not just early-return if `epIndex == kEmberInvalidEndpointIndex` and outdent?

Reply:
Either way, there is the same amount of conditions and no added code or additional call if `epIndex == kEmberInvalidEndpointIndex`

Just a preference I guess, I can do it as you suggest. It doesn't bother me